src/compiler/scala/tools/nsc/settings/Warnings.scala

```diff
@@ -19,12 +19,14 @@ package settings
 trait Warnings {
   self: MutableSettings =>
 
+  val Whelp         = BooleanSetting("-W", "Print a synopsis of warning options.")
+
   // Warning semantics.
-  val fatalWarnings = BooleanSetting("-Xfatal-warnings", "Fail the compilation if there are any warnings.")
+  val fatalWarnings = BooleanSetting("-Werror", "Fail the compilation if there are any warnings.") withAbbreviation "-Xfatal-warnings"
```

Comment:
✅ on maintaining `-Xfatal-warnings` as an alias. I haven't checked throughout, but it would be good to list the settings that have been renamed (if Any) without leaving an alias in place, for future reference.